posts/d3-tutorial-points-axes.md

```diff
@@ -0,0 +1,139 @@
+---
+authors:
+- Justin Risch
+tags:
+- d3.js
+- tutorial
+- data governance
+- data visualization
+date: 2017-08-17T18:22:50.000Z
+title: "Plotting Points on Two Axes with D3.JS"
```

Comment:
The title should be the same as the file name but not sure how to format the period.
Maybe `plotting-points-on-two-axes-with-d3-js.md`?